ckanext/switzerland/templates/showcase/snippets/showcase_item_small.html

```diff
@@ -0,0 +1,53 @@
+{#
+Displays a single dataset of type 'showcase'.
+
+package        - A package to display.
+item_class     - The class name to use on the list item.
+truncate       - The length to trucate the description to (default: 180)
+truncate_title - The length to truncate the title to (default: 80).
+show_remove    - If True, show the remove button to remove showcase/dataset association.
+
+#}
+{% set truncate = truncate or 180 %}
+{% set truncate_title = truncate_title or 80 %}
+{% set title = package.title or package.name %}
+{% set notes = h.markdown_extract(package.notes, extract_length=truncate) %}
+
+{% block package_item %}
+
+<li class="media-item" style="width: 33%; float: left">
```

Comment:
This is the only change to this file in comparison with `showcase/snippets/showcase_item_small.html`. I remembered that @stefina spent a lot of time trying to make organization tiles look nicer in another project and ended up having to do the same thing, so I just jumped to her solution :sweat_smile:

Reply:
Okay, I trust you on this and will merge it.